projects/Mallard/src/navigation/navigators/underlay.tsx

```diff
@@ -26,10 +39,7 @@ const overlayStyles = StyleSheet.create({
     },
 })
 
-const addStaticRouterWithOverlay: NavigatorWrapper = (
-    navigator,
-    getPosition,
-) => {
+const addViewsForTopLayer: NavigatorWrapper = (navigator, getPosition) => {
```

Comment:
i renamed all of these because jesus christ i had no idea what was going on before, hoping the new names are more sensible